pom.xml

```diff
@@ -306,6 +306,9 @@
     <jboss.logging.annotations.version>3.0.3.Final</jboss.logging.annotations.version>
     <jboss.logging.processor.version>3.0.3.Final</jboss.logging.processor.version>
     <jboss.logging.version>3.6.1.Final</jboss.logging.version>
+    <jdk.disable.virtualthreads>[17,20}</jdk.disable.virtualthreads>
+    <jdk.enable.foreign>[22,}</jdk.enable.foreign>
```

Comment:
Maven open ranges do not use braces `}`, but parens `)`: https://maven.apache.org/enforcer/enforcer-rules/versionRanges.html

Reply:
Yup, already noticed that -> https://github.com/jetty/jetty.project/pull/13130#issuecomment-2877653814